test/MUXControlsTestApp/verification/NavigationViewAuto-5.xml

```diff
@@ -1639,7 +1639,7 @@
                                       Visibility=Visible
                                       RenderSize=0,0
                                   [Windows.UI.Xaml.Controls.Primitives.RepeatButton]
-                                      Padding=11,5,11,5
+                                      Padding=0,4,0,0
```

Comment:
I don't understand this either. 
Why is sometimes a 11 replaced with 4 and sometimes a 5 replaced with 4?

I would expect 11,5,11,5 to always become 4,0,0,0 or 0,0,4,0.  Do you have an explanation for that?  Thanks.

Reply:
The root reason seems to be that in the original version no matter what orientation the scrollbar has, the padding for the carat is "11,5,11,5" while in the new version if the scrollbar is horizontal then the padding is "4,0,0,0" (left/decrease) or "0,0,4,0" (right/increase) otherwise the padding is "0,4,0,0" (up/decrease) or "0,0,4,0" (down/increase) , I don't know why the original version had padding at all if none is defined